js_src/pid_display.js

```diff
@@ -88,9 +89,8 @@ app.MessageField.prototype.createDom = function() {
 app.MessageField.prototype.enterDocument = function() {
   app.MessageField.superClass_.enterDocument.call(this);
 
-  var tt = (
-    'Type: ' + this._field_info['type'] + '<br>' +
-    'Name: ' + this._field_info['name'] + '<br>');
+  var tt = 'Type: ' + this._field_info['type'] + '<br>';
+  tt += 'Name: ' + this._field_info['name'] + '<br>';
```

Comment:
I don't know how good the optimiser is, but this would be slightly better if it was just one concat of strings across two lines I think, or does it then throw an error and is that what the brackets were for?